src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -4514,7 +4514,7 @@ public void VerifyMSBuildLogsAMessageWhenLocalPropertyCannotOverrideValueOfGloba
         [Fact]
         public void VerifyPropertyTrackingLoggingDefault()
         {
-            // Having nothing defined should default to only logging reassignments
```

Comment:
I'd make this test a Theory to indeed assert that reassignments get logged when it's turned on.

Reply:
Oh, nevermind, next test does exactly this.